spec/spec_helper.rb

```diff
@@ -1,7 +1,6 @@
 require 'coveralls'
 Coveralls.wear!('rails')
-
-
+require 'support/user_sign_in_helper.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.